Sources/FluentMongoDriver/MongoDB+Schema.swift

```diff
@@ -38,7 +38,7 @@ extension FluentMongoDatabase {
                     }
 
                     var index = CreateIndexes.Index(
-                        named: "unique",
+                        named: name ?? "unique",
```

Comment:
The `.compositeIdentifier(_:)` case will almost always come from this logic in `SchemaBuilder`:

```swift
    public func compositeIdentifier(over fields: FieldKey...) -> Self {
        self.constraint(.constraint(.compositeIdentifier(fields.map { .key($0) }), name: ""))
    }
```
I don't know whether MongoDB can handle an explicitly empty index name, but I imagine it'll probably come out wrong one way or another. Fluent's (admittedly imperfect) assumption here was that for both `.identifier(auto:)` and `.compositeIdentifier(_:)` a constraint name would have to be ignored anyhow (in all three SQL drivers both translate as `PRIMARY KEY`, and in SQLite and MySQL a primary key can't be given a non-default name).

Reply:
If that's always an empty name, I'm sure that will cause issues. Thanks!